### DISCUSSION TITLE
Missing padding around BoleanGroup `No data` peek view

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#6182] [Ideas & Feature Requests]
- Nova Version: 4.32.11

### Description:
I created a BooleanGroup like so:
```
BooleanGroup::make(__('Environments'), 'environments')->options(
    Environment::pluck('name', 'key')
)->hideFalseValues(),
```
When nothing is selected, when I click the "View" button in the Details view, it results in this.
There should be a little padding around the text I believe.
<img width="299" alt="Screenshot 2024-02-05 at 10 18 39 AM" src="https://github.com/laravel/nova-issues/assets/17392251/9f74f339-ce9f-4e0c-b7bd-ff609ceee07b">

When there is a value, there is a py-3, px-4:
![Screenshot 2024-02-05 at 11 13 28 AM](https://github.com/laravel/nova-issues/assets/17392251/4ae2da7f-60f8-4cdb-b7e6-6ef0c22c197c)

**Comment by mrleblanc101** [comment]
For Tag fields, the padding is p-2 all around, not consistent.
Maybe this could be made uniform at the same time.
![Screenshot 2024-02-05 at 11 13 36 AM](https://github.com/laravel/nova-issues/assets/17392251/3ce077a0-daec-4ad0-a652-a2869ac5b145)

**Comment by mrleblanc101** [comment]
The Tag field simply display `--` instead of a "View" button with a "No data" content.
Both Tag and BooleanGroup should probably use `--`, then the padding would matter.
![Screenshot 2024-02-05 at 11 15 42 AM](https://github.com/laravel/nova-issues/assets/17392251/012769e2-a511-4197-96ad-e742ced710d2)

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
